server/container/guice/jpa-guice/src/main/java/org/apache/james/JPAJamesServerMain.java

```diff
@@ -45,7 +46,8 @@ public static void main(String[] args) throws Exception {
         GuiceJamesServerImpl server = new GuiceJamesServerImpl()
                     .combineWith(jpaServerModule, 
                             new JMXServerModule(), 
-                            new LuceneSearchMailboxModule());
+                            new LuceneSearchMailboxModule())
+                    .overrideWith(new NoWebAdminFilterModule());
```

Comment:
No JMAP on top of JPA => no JMAP

I don't want to have dilemna of "should we create a separate config file for jwt" so I decided to not some it during barcamp and keep for later.